VocalRaidAssistant-BCC.toc

```diff
@@ -0,0 +1,20 @@
+## Interface-BCC: 20504
```

Comment:
Can be merged into one .toc file, see https://github.com/WeakAuras/WeakAuras2/blob/main/WeakAuras/WeakAuras.toc

Reply:
I know about using one .toc file for the different interface versions, but as I had created separate files this was not an option. If we have no files that needs to be separate then yes, having one .toc is of course the right choice!